packages/preferences-persistence/src/migrations/preferences-package-data/convert-editor-settings.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Internal dependencies
+ */
+
+export default function convertEditorSettings( data ) {
+	let newData = data;
+	const settingsToMoveToCore = [ 'allowRightClickOverrides' ];
+
+	settingsToMoveToCore.forEach( ( setting ) => {
+		if ( data?.[ 'core/edit-post' ]?.[ setting ] !== undefined ) {
```

Comment:
Should we do the same for `core/edit-site` store? Maybe first check if it's already set in `core` scope..

Reply:
absence of "core/edit-post" value doesn't really mean we don't have a preference, it just means that it was not explicitly set. So I think we should just keep it simple as is.